cvat/apps/engine/serializers.py

```diff
@@ -391,7 +391,7 @@ class Meta:
             'bug_tracker', 'created_date', 'updated_date', 'overlap',
             'segment_size', 'status', 'labels', 'segments',
             'data_chunk_size', 'data_compressed_chunk_type', 'data_original_chunk_type', 'size', 'image_quality',
-            'data', 'dimension', 'subset')
+            'data', 'dimension', 'subset', 'allowed_annotation_mode')
```

Comment:
`allowed_annotation_instrument` maybe?

Reply:
am I right, that for each job type there is always only one instrument?

Reply:
Not one instrument but one group of instruments. But actually, Rectangle is being used for drawing bounding boxes, Tag is for Tag annotation (binary or multiple choise),
Polyshape for drawing polygons (Polylines, polygons etc.).